/usr/local/include/zmq_utils.h:41:32: error: unknown warning group '-Werror', ignored
      [-Werror,-Wunknown-warning-option]
      #pragma GCC diagnostic ignored "-Werror"

Index: include/zmq_utils.h
--- include/zmq_utils.h.orig
+++ include/zmq_utils.h
@@ -39,7 +39,6 @@
 #if defined(__GNUC__) || defined(__GNUG__)
 #pragma GCC diagnostic push
 #pragma GCC diagnostic warning "-Wcpp"
-#pragma GCC diagnostic ignored "-Werror"
 #pragma GCC diagnostic ignored "-Wall"
 #endif
 #pragma message(                                                               \
